Slim/Web/Pages/Search.pm

```diff
@@ -577,6 +592,8 @@ sub fillInSearchResults {
 	# This is very similar to a loop in Slim::Web::Pages::BrowseDB....
 	while (my $obj = $rs->next) {
 
+		$obj->store_column('work' => $obj->work->get_column('title')) if $type eq 'track' && $obj->work->get_column('title');
```

Comment:
I added works as a related result set to tracks, because of a bug I found in the "Songs" results if WORK was included in Title Format - it was showing the work_id rather than name. This line is then needed so that the Track object passed into TitleFormatter has the work name in the expected place. 

Don't know if this is the best/right way to do this!